<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/qspeedtest/icons.ico</normaloff>:/qspeedtest/icons.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>18</number>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditResults">
        <property name="font">
         <font>
          <family>Courier</family>
         </font>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>228</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStartStop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonExit">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxLog"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSaveReport">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Save report as...</string>
     </property>
     <addaction name="actionSavePlainText"/>
     <addaction name="actionSaveBbCode"/>
     <addaction name="actionSaveHtml"/>
    </widget>
    <addaction name="actionLoadHostlist"/>
    <addaction name="separator"/>
    <addaction name="menuSaveReport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuCopyReport">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Copy report to clipboard as...</string>
     </property>
     <addaction name="actionCopyPlainText"/>
     <addaction name="actionCopyBbCode"/>
     <addaction name="actionCopyHtml"/>
    </widget>
    <addaction name="menuCopyReport"/>
    <addaction name="separator"/>
    <addaction name="actionCopyLog"/>
    <addaction name="actionClearLog"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpContents"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAboutQSpeedTest"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLoadHostlist">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load hostlist from file...</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionSavePlainText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Plain text file</string>
   </property>
  </action>
  <action name="actionSaveBbCode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>bbCode file</string>
   </property>
  </action>
  <action name="actionSaveHtml">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>HTML file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="actionCopyPlainText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Plain text</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionCopyBbCode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>bbCode</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionCopyHtml">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>HTML</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAboutQSpeedTest">
   <property name="text">
    <string>About QSpeedTest</string>
   </property>
   <property name="toolTip">
    <string>About QSpeedTest</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionHelpContents">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Contents</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionCopyLog">
   <property name="text">
    <string>Copy log to clipboard</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionClearLog">
   <property name="text">
    <string>Clear log</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
